app.R

```diff
@@ -18,30 +22,58 @@ ui <- fluidPage(
     # Sidebar with a slider input for number of bins
     sidebarLayout(
         sidebarPanel(
+            checkboxGroupInput(inputId = "stems_plot",
+                               label = "Census year:",
+                               choices = c("2008-2010" = "stems_plot$CensusID == 1",
+                                 "2013" = "stems_plot$CensusID == 2",
+                                 "2018" = "stems_plot$CensusID == 3")
+            ),
             sliderInput("bins",
                         "Number of bins:",
                         min = 1,
-                        max = 50,
-                        value = 30)
-        ),
+                        max = 30,
+                        value = 15)
+            ),
 
         # Show a plot of the generated distribution
         mainPanel(
-           plotOutput("distPlot")
-        )
+           plotOutput("histPlot")
+        ))
     )
-)
+
 
 # Define server logic required to draw a histogram
 server <- function(input, output) {
 
-    output$distPlot <- renderPlot({
+    stem1 <- here::here("data/scbi.stem1.csv") %>%
+        read_csv()
+    stem2 <- here::here("data/scbi.stem2.csv") %>%
+        read_csv()
+    stem3 <- here::here("data/scbi.stem3.csv") %>%
+        read_csv()
+
+    stems <- rbind(stem1, stem2, stem3)
+
+    stems_plot <- stems %>%
+        filter(dbh != "NULL") %>%
+        na.omit(stems) %>%
+        mutate(CensusID = as.factor(CensusID),
+               dbh = as.numeric(dbh))
```

Comment:
b/c of a few outlier dbh values, this histogram is ultra right-skewed. Try dropping the few ultra large outliers

Reply:
You can also do the filtering of censuses here